Libplanet/Action/PolymorphicAction.cs

```diff
@@ -266,11 +266,6 @@ private static Type GetType(string typeId)
                 types = new Dictionary<string, Type>();
                 foreach (Assembly a in Assemblies)
                 {
-                    if (!(a is { } asm))
```

Comment:
This deletion will change semantics when `Assemblies` contains a null value. if `asm` is a matter, we can omit it.

Reply:
`Assembly a` has not implemented in the file.